CONTRIBUTORS

```diff
@@ -20,20 +29,30 @@ Thank you!
     Alex Dowad <alexinbeijing@gmail.com>
     Alexey Veselovsky <alexey.veselovsky@eykontech.com>
     Alexis Robert <alexis.robert@gmail.com>
+    Alex Rousskov
+    Alex Rousskov <>
+    Alex Rousskov <rousskov@measurement-factory.com
     Alex Rousskov <rousskov@measurement-factory.com>
     Alex Rousskov <rousskov@squid-cache.org>
     Alex Wu <alex_wu2012@hotmail.com>
     Alin Nastac <mrness@gentoo.org>
     Alter <alter@alter.org.ua>
+    Ambrose Li <ambrose.li@gmail.com>
+    Amish <3330468+amishmm@users.noreply.github.com>
+    Amish <amishxda@users.noreply.github.com>
     Amit Klein <amit.klein@safebreach.com>
+    amos@georgia.treenetnz.com <>
     Amos Jeffries
+    Amos Jeffries <>
```

Comment:
I have been trying to do that where possible. These empty email brackets have been causing me some trouble with sed patterns and shell syntax.

I have also been contemplating an awk script that assembles an array or name+email to remove those dupes. If you have a good idea please go ahead.

Reply:
> Would you be willing to polish the script so that it automatically removes obvious duplicates? If not, I will do that.

Done at 9e59207. Resolving this change request. I will describe recent branch changes separately.